pkg/odo/cli/project/create.go

```diff
@@ -53,6 +57,29 @@ func (pco *ProjectCreateOptions) Validate() (err error) {
 
 // Run runs the project create command
 func (pco *ProjectCreateOptions) Run() (err error) {
+
+	// Machine readable output
+	if pco.OutputFlag == "json" {
+
+		// Create the project
+		err = project.Create(pco.Client, pco.projectName, false)
```

Comment:
what if someone wants to wait and get a machine readable output as well?

Reply:
What do you mean? This method is done the exact same way in `odo app list` as well in terms of output, etc. We should create the project within the `if` statement rather than outside.